src/components/ui-shell/platform-side-nav-fixed.hbs

```diff
@@ -0,0 +1,39 @@
+<aside class="bx--side-nav bx--side-nav--expanded bx--side-nav bx--side-nav--fixed">
+  <div class="bx--side-nav__content">
+    {{> platform-side-nav-title-bar sidenav=sidenav }}
+    <div class="bx--side-nav__scroll">
+      <nav class="bx--side-nav__nav">
+        <ul class="bx--side-nav__nav-items">
+          {{#each sidenav.links}}
+          <li class="bx--side-nav__nav-item">
+            {{#if this.active}}
+            <div class="bx--side-nav__category bx--side-nav__category--active">
+            {{else}}
+            <div class="bx--side-nav__category">
+            {{/if}}
+              <div class="bx--side-nav__category-header">
+                <div class="bx--side-nav__category-title title--fixed">
+                  {{ this.category }}
+                </div>
+              </div>
+              <ul class="bx--side-nav__category-itms">
+                {{#each this.links}}
+                {{#if this.active}}
```

Comment:
If that conditional is indented then the `<li>`'s would be indented incorrectly, right? (4 spaces instead of 2). That said I totally think indenting  makes it much more readable.

Reply:
Good thoughts - I'd be inclined to take Handlebars directives into account for indent (instead of just looking at HTML tags), but I see pros/cons of either side. We can take whichever that makes sense for this specific here.